extide/gradle/src/org/netbeans/modules/gradle/customizer/GradleExecutionPanel.java

```diff
@@ -54,9 +62,32 @@ public GradleExecutionPanel(Project project) {
         GradleBaseProject gbp = GradleBaseProject.get(project);
         if (gbp != null) {
             lbReadOnly.setVisible(!gbp.isRoot());
-            cbTrustProject.setEnabled(gbp.isRoot());
+            lbTrustLevel.setEnabled(gbp.isRoot());
+            cbTrustLevel.setEnabled(gbp.isRoot());
             lbTrustTerms.setEnabled(gbp.isRoot());
-            cbTrustProject.setSelected(ProjectTrust.getDefault().isTrusted(project));
+            
+            if (ProjectTrust.getDefault().isTrustedPermanetly(project)) {
+                cbTrustLevel.setSelectedItem(LEVEL_PERMANENT);
+            } else if (ProjectTrust.getDefault().isTrusted(project)) {
+                cbTrustLevel.setSelectedItem(LEVEL_TEMPORARY);
+            } else {
+                cbTrustLevel.setSelectedItem(LEVEL_NONE);
+            }
+            
+            cbTrustLevel.setRenderer(new DefaultListCellRenderer() {
+                @Override
+                public Component getListCellRendererComponent(JList<?> list, Object value, int index, boolean isSelected, boolean cellHasFocus) {
+                    if (value == null) {
+                        value = ""; // NOI18N
+                    } else {
+                        try {
+                            value = NbBundle.getMessage(GradleExecutionPanel.class, "GradleExecutionPanel.cbTrustLevel." + value.toString()); // NOI18N
```

Comment:
If you add this line into the Level enum toString() method, then you do not need a new cell renderer.

Reply:
Fixed in 7e035b7